cmd/containerd-shim/shim_linux.go

```diff
@@ -45,3 +44,8 @@ func setupSignals() (chan os.Signal, error) {
 func newServer() (*ttrpc.Server, error) {
 	return ttrpc.NewServer(ttrpc.WithServerHandshaker(ttrpc.UnixSocketRequireSameUser()))
 }
+
+// setSubreaper sets the value i as the subreaper setting for the calling process
+func setSubreaper(i int) error {
```

Comment:
Guess I can even remove the argument, as it's a fixed value; let me update

Reply:
please check here https://github.com/containerd/containerd/blob/master/sys/reaper_linux.go#L36